jqwik-mockito/build.gradle.kts

```diff
@@ -0,0 +1,153 @@
+    api("net.jqwik:jqwik:${jqwikVersion}")
+    compileOnly("org.mockito:mockito-core:${mockitoVersion}")
+    compileOnly("org.mockito:mockito-junit-jupiter:${mockitoVersion}")
+
+    testCompileOnly("org.projectlombok:lombok:${lombokVersion}")
+    testAnnotationProcessor("org.projectlombok:lombok:${lombokVersion}")
+
+    testImplementation("org.junit.jupiter:junit-jupiter:${junitJupiterVersion}")
+    testImplementation("org.assertj:assertj-core:${assertJVersion}")
+    testImplementation("org.mockito:mockito-junit-jupiter:${mockitoVersion}")
+}
+
+publishing {
+    repositories {
+        maven {
+            // hint: password is in ~/.gradle/gradle.properties
+            val ossrhUsername: String = project.findProperty("ossrhUsername") as? String ?: ""
+            val ossrhPassword: String = project.findProperty("ossrhPassword") as? String ?: ""
+
+            credentials {
+                username = ossrhUsername
+                password = ossrhPassword
+            }
+
+            // change URLs to point to your repos, e.g. http://my.org/repo
+            val releasesRepoUrl = "https://s01.oss.sonatype.org/service/local/staging/deploy/maven2/"
+            val snapshotsRepoUrl = "https://s01.oss.sonatype.org/content/repositories/snapshots/"
+            url = uri(if (isSnapshotRelease) { snapshotsRepoUrl } else { releasesRepoUrl })
+        }
+    }
+    publications {
+        create<MavenPublication>("jqwikMockito") {
+            groupId = moduleGroupId
+            artifactId = artifactId
+            from(components["java"])
+            pom {
+                groupId = moduleGroupId
+                name = artifactId
+                description = "Jqwik Mockito support module"
+                url = "https://github.org/$githubProjectName/$artifactId"
+                licenses {
+                    license {
+                        name = "Eclipse Public License - v 2.0"
+                        url = "http://www.eclipse.org/legal/epl-v20.html"
+                    }
+                }
+                developers {
+                    developer {
+                        id = "agustafson-atl"
+                        name = "Andrew Gustafson"
+                        email = "agustafson@atlassian.com"
+                    }
+                    developer {
+                        id = githubProjectName
+                        name = "Johannes Link"
+                        email = "business@johanneslink.net"
+                    }
+                }
+                scm {
+                    connection = "scm:git:git://github.com/$githubProjectName/$artifactId.git"
+                    developerConnection = "scm:git:git://github.com/$githubProjectName/$artifactId.git"
+                    url = "https://github.com/$githubProjectName/$artifactId"
+                }
+            }
+        }
+    }
+}
+
+signing {
+    if (!isSnapshotRelease) {
+        sign(publishing.publications["jqwikMockito"])
+    }
+}
+
+//wrapper {
```

Comment:
Not sure how to get this to work in a Kotlin build file, and not sure if it's necessary. Will look at how I can get it to work